app/views/reports/forms_with_csv_submission_enabled.html.erb

```diff
@@ -0,0 +1,29 @@
+<% set_page_title(t(".heading"))%>
+<% content_for :back_link, govuk_back_link_to(report_features_path, t("reports.back_to_feature_usage")) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l"><%= t(".heading")%></h1>
+
+    <p><%=govuk_link_to(t(".download_csv"), report_live_forms_with_csv_submission_enabled_csv_path)%></p>
+  </div>
+
+  <div class="govuk-grid-column-full">
+    <%= govuk_table do |table| %>
+
+      <%= table.with_head do |head| %>
+        <%= head.with_row do |row| %>
+          <%= row.with_cell(text: t("reports.form_or_questions_list_table.headings.form_name")) %>
+          <%= row.with_cell(text: t("reports.form_or_questions_list_table.headings.organisation")) %>
+        <% end %>
+      <% end %>
+      <%= table.with_body do |body| %>
+        <% forms.each do |form| %>
+          <%= body.with_row do |row| %>
+            <%= row.with_cell(text: govuk_link_to(form[:form_name], live_form_pages_path(form_id: form[:form_id]))) %>
```

Comment:
I think `live_form_pages_path` makes sense for the list of forms with routes (since routes are displayed on that page), but for the other reports `live_form_path` might make more sense

```suggestion
            <%= row.with_cell(text: govuk_link_to(form[:form_name], live_form_path(form_id: form[:form_id]))) %>
```

Reply:
Ah yeh good point! It makes more sense to see the form settings.